src/ochubryk/hometask3/couples_of_elements.py

```diff
@@ -0,0 +1,15 @@
+# Дан список чисел. Посчитайте, сколько в нем пар элементов,
+# равных друг другу. Считается, что любые два элемента, равные
+# друг другу образуют одну пару, которую необходимо посчитать.
+# Входные данные - строка из чисел, разделенная пробелами.
+# Выходные данные - количество пар.
+# Важно: 1 1 1 - это 3 пары, 1 1 1 1 - это 6 пар.
+
+initial_string = [int(number) for number
+                  in input('Введите строку из чисел: ').split()]
+summary = 0
+
+for number in initial_string:
+    summary += initial_string.count(number) - 1
+
+print(int(summary / 2))
```

Comment:
this / 2 just shows we have more optimal way )))) 
if we have 1 1 1 you will count 2 + 2 + 2 and in fact you have 2 opportunities for first 1 for second and 0 for third one. it would be arithmetic progression with d=-1 and sum = 3(n)*(a0+a2)/2 = 3*(2+0)/2. You are just counting not average and that's why dividing in the last print.
In fact we have other formulas for this task :)

Reply:
Your suggestions about an arithmetic progression were really great so I used it in two ways :)